javascript-objects/main.js

```diff
@@ -0,0 +1,36 @@
+var student = {
+  firstName: 'Nick',
+  lastName: 'Newton',
+  age: 31
+};
+
+var fullName = student.firstName + ' ' + student.lastName;
+console.log('value of fullName:', fullName);
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Regional Sales Manager';
+console.log('value of livesInIrvine:', student.livesInIrvine);
+console.log('value of previousOccupation:', student.previousOccupation);
+console.log('value of student', student);
+
+var vehicle = {
+  make: 'Honda',
+  model: 'Civic',
+  year: 2018
+};
+
+vehicle.color = ['red'];
+vehicle.isConvertible = [false];
```

Comment:
This is not correct, please review the syntax for accessing object properties with square bracket notation.

Reply:
If done correctly, the square brackets will end up being used on the left-hand side of the assignment operator.